core/src/por/reducer.ts

```diff
@@ -0,0 +1,71 @@
+import { PorError, PorErrorCode } from './errors'
+
+export const DATA_FEED_REDUCER_MAPPING = {
```

Comment:
Can't we just import the reducer, rather than copy it?

Reply:
would it be okay to import reducer.ts from `./src/worker/reducer.ts`? thought por can't refer from `worker` foler